.github/workflows/ci_release.yml

```diff
@@ -19,6 +22,9 @@ on:
           - patch
           - minor
           - major
+      tag-as:
```

Comment:
Input for manual trigger. Basically this case:
<img width="359" alt="image" src="https://github.com/user-attachments/assets/9b7d20d9-133b-4915-80b0-098b35c9bc14">